### DISCUSSION TITLE
No idToken being returned

### DISCUSSION THREAD

**Discussion by briankostar** [Web3Auth/Web3Auth#361] [Help]
Trying to login with Google -- after logging in, and doing web3auth.getUserInfo(), there is no tokenId. 
This is a problem as it as we cant verify the user on the backend without the tokenId. 

Issue is only with 1 particular google account after setting up 2FA. Other accounts are still ok. 

Clearing cookies, localstorage, and logging in from incognito show same issue.

<img width="1669" alt="Screen Shot 2022-08-02 at 3 14 45 PM" src="https://user-images.githubusercontent.com/4313671/182455688-b90918d2-b2a7-40d4-aa59-5d668e934389.png">

**Comment by YZhenY** [comment]
@arch1995 can you loook into this

**Comment by shahbaz17** [comment]
Hey @briankostar 

Which version of `web3auth` are you using?

ClientId is available from v1.1.1, but always try using the latest version.

**Reply by briankostar** [reply]
yup on 1.1.4

**Comment by himanshuchawla009** [comment]
@briankostar we have created an issue for you here: https://github.com/Web3Auth/Web3Auth/issues/362

can you plz check if any network call is failing or if any other errors are being logged in the console, If yes plz reply on the issue link.

**Comment by himanshuchawla009** [comment]
Hey @briankostar, we have fixed this issue. Please try again, you should be able to get idToken now. Let us know if you are still facing it.
Thanks for reporting the issue.

**Reply by briankostar** [reply]
Confirming that its working now, Thanks @himanshuchawla009 !
